shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -395,19 +410,24 @@ @implementation ForwardingGestureRecognizer {
   // So this is safe as when FlutterView is deallocated the reference to ForwardingGestureRecognizer
   // will go away.
   UIView* _flutterView;
+  // Counting the touches that has started in one touch sequence.
+  NSInteger _currentEventTouchCount;
```

Comment:
nit: maybe call it pointersCount? (I believe this is counting the number of pointers that are currently touching the screen?)